source/hardware/controllers/numark_mixtrack_pro.rst

```diff
@@ -28,65 +28,89 @@ Mapping Description
 4.  **CHANNEL BASS:** Adjusts the low (bass) frequencies of the corresponding channel.
 5.  **CUE:** Sends pre-fader audio to the Cue Channel for headphone monitoring.
 6.  **CUE GAIN:** Adjusts the audio level of the Cue channel.
-7.  **CUE MIX:** Turn to mix between Cue and Program in the Headphone channel. When all the way to the left, only channels routed to CUE will be heard. When all the way right, only the Program mix
+7.  **CUE MIX:** Turn to mix between Cue and Program in the Headphone channel. When all the way to the left, only channels routed to Cue will be heard. When all the way right, only the Program mix
     will be heard.
 8.  **BROWSE KNOB:** Turn this knob to scroll through lists of tracks / library. When a file is selected, press the knob to load the currently highlighted song into the first stopped deck.
-9.  **BACK:** This button will switch the Browse Knob to scroll into the track list (FILE LED will illuminate) or into the library section (FOLDER LED will illuminate).
-10. **LOAD A / LOAD B:** Press one of these buttons while a track is selected to assign it to Deck A or Deck B, respectively.
-    **\*Only if the track is paused. Put the pitch in 0% at load**\ *(without mattering where the slider is)*.
+9.  **BACK:** This button will switch the Browse Knob to scroll into the track list (File LED will illuminate) or into the library section (Folder LED will illuminate).
+10. **LOAD A/B:** When a deck is paused and a track is selected, press one of these buttons to load the selected track into the respective deck.
+
+    * *Note:* The pitch is set to 0% at load.
+
```

Comment:
Why is this? Does this not cause issues with sync lock?

Reply:
This is unusual mainly because the hardware pitch faders get out of sync.